CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 ### Improvements
 
 - Added cleanup logic at interpreter shutdown to close all active sessions.
+- Replaced all pandas text to lowercase.
```

Comment:
I don't think we need to include this entry in the changelog.

Reply:
I can remove it if it is not needed.